lineman/app/components/dashboard_page/dashboard_page_group_collection.haml

```diff
@@ -0,0 +1,9 @@
+%section.dashboard-page-threads-container{ng-show: 'any()', role: 'region', aria-label: "{{ name | translate }}"}
+  .dashboard-page-group-name{aria-hidden: 'true'}
+    %img.selector-list-item-group-logo.pull-left{src: "{{group.logoUrl()}}", aria-hidden: 'true'}>
+    %h2.dashboard-page-group-name
+      %a{href: '/g/{{group.key}}'} {{group.name}}
```

Comment:
This is an accessibility anti-pattern that I introduced that I would like to squelch ASAP. Hiding the H2 was a supremely dumb decision I made while my brain was on the shelf. A better version (needs test) would be:

```
%section.dashboard-page-threads-container{ng-show: 'any()', role: 'region', aria-labelledby: "{{ name }}"}
  .dashboard-page-group-name
    %img.selector-list-item-group-logo.pull-left{src: "{{group.logoUrl()}}", aria-hidden: 'true'}>
    %h2.dashboard-page-group-name{id: '{{ name }}'}
      %a{href: '/g/{{group.key}}'} {{group.name}}

```

[See here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) for explanation of aria-labelledby.